### PR TITLE
Malware Protection Essentials - ASIM Based Solution

### PR DESCRIPTION
**Pull request by rahul0216**
Change(s):
   - Added Analytic Rule, Hunting Queries, Workbook

   Reason for Change(s):
   - Initial Commit

   Version Updated:
   - Yes

   Testing Completed:
   - Yes
   
[Malware.docx](https://github.com/Azure/Azure-Sentinel/files/13939028/Malware.docx)

### CODE REVIEW

File: Solutions/Malware Protection Essentials/Analytic Rules/BackupDeletionDetected.yaml [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+  - connectorId: SentinelOne
+    dataTypes:
+      - SentinelOne_CL
+  - connectorId: VMwareCarbonBlack
+    dataTypes:
+      - CarbonBlackNotifications_CL
+  - connectorId: CiscoSecureEndpoint
+    dataTypes:
+      - CiscoSecureEndpoint_CL
+  - connectorId: TrendMicroApexOne
```
Comment (manishkumar1991) [inline Solutions/Malware Protection Essentials/Analytic Rules/BackupDeletionDetected.yaml:27]: Kindly consider addition of new trend micro AMA data connector id as well.

File: Solutions/Malware Protection Essentials/ReleaseNotes.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+| **Version** | **Date Modified (DD-MM-YYYY)** | **Change History** |
+|-------------|--------------------------------|--------------------|
+|3.0.0        |12-21-2023                      |Initial Release     |
```
Comment (manishkumar1991) [inline Solutions/Malware Protection Essentials/ReleaseNotes.md:3]: Can check with PM, but as per me, in change history, we should be adding all the details of what we are bringing in this solution.

File: Solutions/Malware Protection Essentials/Analytic Rules/BackupDeletionDetected.yaml (hunk 1 above)
Comment (manishkumar1991) [reply Solutions/Malware Protection Essentials/Analytic Rules/BackupDeletionDetected.yaml:27]: Please consider this in all activity and hunting rules

**Comment by manishkumar1991** [comment]
Requesting for a read me file here, which will guide customer on how useful this solution is, and some screenshot which will tell customer exactly what will be deployed before installing the solution , Or one video would be nice addition .

File: Solutions/Malware Protection Essentials/Analytic Rules/BackupDeletionDetected.yaml (hunk 1 above)
Comment (rahul0216) [reply Solutions/Malware Protection Essentials/Analytic Rules/BackupDeletionDetected.yaml:27]: Added TrendMicroApexAma connector in all hunting queries and analytic rules.

File: Solutions/Malware Protection Essentials/ReleaseNotes.md (hunk 2 above)
Comment (rahul0216) [reply Solutions/Malware Protection Essentials/ReleaseNotes.md:3]: This is to capture the changes history, this is fine. We'll write a blog post for solution details.

**Comment by rahul0216** [comment]
@manishkumar1991 This is great suggestion. We will write a blog post explaining about new solution.

**Review by manishkumar1991 — APPROVED** [review]
Approved

**Comment by sandeep5234** [comment]
Hi When this will be approved and available in master branch?

**Pull request merged**